### ISSUE TITLE
Transaction creation on the group page

### ISSUE THREAD

**Issue by nmorduch** [littleweaver/django-argus#26]
Not immediately, I imagine, but I would eventually like to get the transaction creation and editing to be done from within the group_detail page. Transaction editing should probably actually be on any page where there is a log of transactions.

**Comment by melinath** [comment]
Technically resolved before 12f4a948b372604ecc80735a2115c2e9064ce8a0 - but do we want to set some defaults if creating on party/category pages?

**Comment by melinath** [comment]
@harrislapiroff we also need to have better form behavior. "Sharers" should only be visible if it's an even split, and the per-member fields should only be visible if it's a manual split. Not to mention that the flow needs a lot of work...

**Comment by nmorduch** [comment]
Oooh yeah set it by default to be in that category or for that party.

@harrislapiroff we'll need to hinge a show/hide on which piece of a select is selected, which sounds complicated form-wise.

**Comment by melinath** [comment]
@nmorduch - _for_ that party? Like... paid _to_ that party? Or paid by that party? Or it depends whether it's a member or not?

**Comment by nmorduch** [comment]
Paid by if a member, paid to if not? Though that does get a bit confusing. Maybe it is a little silly to have that. Maybe actually we should only have transaction creation on the group page?

**Comment by melinath** [comment]
It sounds like you might be concerned that things are magically and confusingly happening... We can still display the whole form, and just have the category/party set by default. (So you'd still be able to change them to something else.) Or we could disable the fields, but still display them.

**Comment by nmorduch** [comment]
Right, but as a user I don't think I'm going to want to input from a person/category page with the idea that this new entry belongs to that person/category. I'm going to be looking at those pages for information, but when I'm inputting things it's probably going to be with a pile of receipts belonging to a few people.

Based on that idea, I think it only makes sense to have transaction creation on every page if it's generally small and collapsed at the top, and feels as unchanging as the header. And then the problem would be if you added something from a specific page but it didn't belong on that page, and then it wouldn't add to the log you could see, so we'd need a notification about it. I think that right now I might prefer it only to be on the main group page.
